### ISSUE TITLE
Incorrect reverse dependencies when a package is required multiple times

### ISSUE THREAD

**Issue by anntzer** [naiquevin/pipdeptree#45]
Within a fresh virtualenv, install `pipdeptree`, and then `dateutil` and `cycler`, which both depend only on `six`.
```
$ python -mpipdeptree -r
six==1.10.0
  - cycler==0.10.0 [requires: six>=1.5] # Oops, this should be dateutil.
six==1.10.0
  - cycler==0.10.0 [requires: six]
```

**Comment by naiquevin** [comment]
This looks like a bug. Thanks for reporting.

**Comment by naiquevin** [comment]
Fix included in the 0.7.0 release on PyPI.
